doc/website/advanced/roudi-command-line-arguments.md

```diff
@@ -0,0 +1,16 @@
+|  -d   | --domain-id         | Unsigned integer                                              | Sets the domain id. No communication can happen between different domains. Experimental feature!                                                                                                                                                     |
+|  -u   | --unique-roudi-id   | Unsigned integer                                              | Sets the unique RouDi identifier                                                                                                                                                                                                                     |
+|  -m   | --monitoring-mode   | String (on, off)                                              | Enables or disables the process alive monitoring feature. If enable RouDi terminates and kills unresponsive process.                                                                                                                                 |
+|  -l   | --log-level         | String (off, fatal, error, warning, info, debug, trace)       | Set the log level. Default log level is 'info'.                                                                                                                                                                                                      |
+|  -x   | --compatibility     | String (off, major, minor, patch, commitId, buildDate)        | Sets the compatibility check level between application and RouDi. Default is 'patch'. This can be useful if old apps are build against and old iceoryx version. Use with care!                                                                       |
+|  -t   | --termination-delay | Unsigned integer                                              | Sets the delay in seconds before RouDi sends SIGTERM to running applications at shutdown. Default is '0'.                                                                                                                                            |
+|  -k   | --kill-delay        | Unsigned integer                                              | Sets the delay in seconds before RouDi sends SIGKILL to application which did not respond to the initial SIGTERM signal. Default is '45'.                                                                                                            |
+|  -c   | --config-file       | String (Absolute filesystem path to a config in YAML format)  | Sets the config file. If option is not given, fallbacks in descending order: 1. /etc/iceoryx/roudi_config.toml 2. hard-coded config. See [configuration guide](configuration-guide.md#dynamic-configuration) for information on the format. |
```

Comment:
TOML, not YAML, or?

Reply:
@budrus Of course you are right! :see_no_evil: